app/portainer/components/endpoint-list/endpoint-item/endpointItem.html

```diff
@@ -38,24 +38,24 @@
         <span class="blocklist-item-desc">
           <span>
             <span style="padding: 0 7px 0 0;">
-              <i class="fa fa-th-list space-right" aria-hidden="true"></i>{{ $ctrl.model.Snapshots[0].StackCount }} stacks
+              <i class="fa fa-th-list space-right" aria-hidden="true"></i>{{ $ctrl.model.Snapshots[0].StackCount }} stack<span ng-if="$ctrl.model.Snapshots[0].StackCount === 0 || $ctrl.model.Snapshots[0].StackCount > 1">s</span>
```

Comment:
```suggestion
              <i class="fa fa-th-list space-right" aria-hidden="true"></i> {{ $ctrl.model.Snapshots[0].StackCount === 0 ? 'stack' : 'stacks' }}
```

Reply:
I think we'll probably need to preserve the `{{ $ctrl.model.Snapshots[0].StackCount }}` bit before the noun in order to print the number of stacks. Also, the ternary operator's condition should perhaps be `$ctrl.model.Snapshots[0].StackCount === 1`? Sorry if I'm being pedantic! :wink:

Reply:
You're right, it was actually more of an example than a suggestion as I did not test it.